src/components/application_manager/include/application_manager/commands/mobile/delete_command_response.h

```diff
@@ -51,7 +51,8 @@ class DeleteCommandResponse : public CommandResponseImpl {
    *
    * @param message Incoming SmartObject message
    **/
-  explicit DeleteCommandResponse(const MessageSharedPtr& message);
+  explicit DeleteCommandResponse(const MessageSharedPtr& message,
```

Comment:
@Kozoriz Remove `explicit`. In ctor with 2 parameters it will not work. Check all commands please!